data/org.freedesktop.UDisks2.xml

```diff
@@ -1344,8 +1344,8 @@
         Gets a file descriptor for the device that is suitable to be
         used for benchmarking the device (transfer rate, access time
         etc.). Note that the file descriptor may be opened with the
-        <literal>O_DIRECT</literal> and <literal>O_SYNC</literal>
-        flags so care must be taken to only perform page-aligned I/O.
+        <literal>O_DIRECT</literal> flag so care must be taken to only perform
+        block-aligned (512 byte aligned) I/O.
```

Comment:
As for the block size, this is what ``man:open(2)`` says:
```
Since Linux 2.6.0, alignment to the logical block size of the underlying storage (typically 512 bytes) suffices.  The logical block size can be determined using the ioctl(2) BLKSSZGET operation...
```

So I really think it is the block size of the device. And udisks should provide this value as a property. If it doesn't, please create an RFE (issue).